Content.Shared/_RMC14/Marines/Squads/SquadSystem.cs

```diff
@@ -59,6 +59,7 @@ public sealed class SquadSystem : EntitySystem
     [Dependency] private readonly IPrototypeManager _prototypes = default!;
     [Dependency] private readonly SharedRMCBanSystem _rmcBan = default!;
     [Dependency] private readonly SharedCMChatSystem _rmcChat = default!;
+    [Dependency] private readonly IEntityManager _entMan = default!;
```

Comment:
You don't need to import entity manager into systems, they already have it and most methods on it have proxy methods on entity systems
```suggestion
```